wagtail/wagtailforms/models.py

```diff
@@ -295,7 +295,15 @@ def process_form_submission(self, form):
 
     def send_mail(self, form):
         addresses = [x.strip() for x in self.to_address.split(',')]
-        content = '\n'.join([x[1].label + ': ' + text_type(form.data.get(x[0])) for x in form.fields.items()])
+        content = []
+        for field in form:
+            value = field.value()
+            if isinstance(value, list):
+                value = ', '.join(value)
+                if len(value) == 0:
+                    value = 'None'
+            content.append('{}: {}'.format(field.label, value))
```

Comment:
Maybe, instead of converting an empty string to 'None', it would be better to only add a line to the email if the value is not empty? This could be done by adding a `if value != '':` before this statement.